README-manual.md

```diff
@@ -14,7 +14,7 @@ However, different from the default react native plugin, this plugin needs to ha
 
 ```ts
 import { ApplicationInsights } from '@microsoft/applicationinsights-web';
-
+import {ReactNativeManualDevicePlugin} from '@microsoft/applicationinsights-react-native/dist-esm/manualIndex';
```

Comment:
We should not be importing from `/dist-esm/manualIndex'` as this causes issues.

nit: there should also be spacing around the import name.

Reply:
Why? Because this may break in a future release when (not if) we start supporting multiple targeted ES specification